Sebastian-Sole/handicappin
# ✨feat: trpc

## Description

- Added tRPC for auth and api calls
- User can add rounds in /rounds/add
- Auth calls are now handled by tRPC

## How to test
- Visit [prod](https://handicappin-git-feat-trpc-sebastiansoles-projects.vercel.app/), and login/signup (github/google er ikke enda implementert)
- Visit [here](https://handicappin-git-feat-trpc-sebastiansoles-projects.vercel.app/rounds/add) to add a round
- Add a round and see that you get redirected to "/" and that a toast appears

## Future Work
- Toasts
- View rounds
